ui/store/actions.js

```diff
@@ -776,6 +776,33 @@ export function updateTransactionSendFlowHistory(txId, sendFlowHistory) {
   };
 }
 
+export function backupUserData() {
+  return async () => {
+    let backedupData;
+    try {
+      backedupData = await promisifiedBackground.backupUserData();
+    } catch (error) {
+      log.error(error.message);
+      throw error;
+    }
+
+    return backedupData;
+  };
+}
+
+export function restoreUserData(jsonString) {
```

Comment:
Nit: The "return a function" pattern is for Redux action creators, but this function is not an action creator. It can be a regular async function, and does not need to be invoked with `dispatch`.